src/components/Observations/ObsStatus.js

```diff
@@ -37,21 +36,17 @@ const ObsStatus = ( {
 
   return (
     <View className={classNames( "flex", flexDirection, classNameMargin )}>
-      <ActivityCount
-        icon="label"
-        margin={margin}
+      <IdentificationsCount
+        classNameMargin={margin}
```

Comment:
shouldn't this and `CommentsCount` have a filled prop which will be filled if the Id or Comment is unread?

Reply:
Yes, both those components do have a `filled` boolean prop ready to be used. I have only concerned myself with the UI for this PR because the issue was about UI only. In the UiLib there is an example of the filled usage. Do you want me to add the logic part to check whether to use the filled UI for an observation here or we make a separate issue for that?

Reply:
I think it either makes sense to leave this component in the UI library & not try to integrate it into `ObsStatus` at all, or it makes sense to have a fully working component in `ObsStatus`.

Reply:
So, I was doing some digging in the git history of the ObsStatus component. From what I can see the boolean used for both comments and identifications together was `observation.viewed` to indicate if it was viewed or not. This is only local I think (not coming from the API). In our case here we would need two values separately for the id and comment.
I also tried to make an API call to `v2/observations/updates?viewed=false&observation_by=owner` which differs in the result values from API v1, returning currently an array like so: `[{"id": 371531736}, {"id": 371518022}]` which is neither the id of the identification nor the comment which was not viewed in this case.

I want to try to not leave unfinished work until the end of the week, so I have removed the usage of the components from ObsStatus and will punt the logic part to a separate issue.

Reply:
But I also think the logic part will not end up in ObsStatus because, e.g., in ObsDetail we are using also ObsStatus and it will always have all comments and identification viewed there.